driver/src/main/scala/com/stratio/crossdata/driver/Driver.scala

```diff
@@ -96,7 +96,8 @@ object Driver {
     newSession(driverConf.setClusterContactPoint(seedNodes))
 
   private[crossdata] def newSession(driverConf: DriverConf, authentication: Authentication): Driver = {
-    val driver = new ClusterClientDriver(driverConf, authentication)
+    //TODO: Enable driver selection by configuration
```

Comment:
Instead of doing the selection by configuration, the user who implements the driver can use the http factory commented below